frontend/src/components/contactsForm.tsx

```diff
@@ -0,0 +1,310 @@
+/* eslint-disable @typescript-eslint/no-misused-promises */
+import React, { useState } from 'react'
+import axios from 'axios'
+
+type ImportantDate = {
+  date: string
+  eventType: string
+}
+
+type Relationship = {
+  relationshipType: string
+}
+
+interface Props {
+  setShowCreateAContactForm: (value: boolean) => void
+  handleContactCreate: (newContact: any) => void
+}
+
+export const CreateAContactForm = ({ setShowCreateAContactForm, handleContactCreate }: Props) => {
+  const [firstName, setFirstName] = useState<string>('')
+  const [lastName, setLastName] = useState<string>('')
+  const [relationships, setRelationships] = useState<Relationship[]>([])
+  const [importantDates, setImportantDates] = useState<ImportantDate[]>([])
+  const [eventDate, setEventDate] = useState<string>('')
+  const [eventType, setEventType] = useState<string>('')
+  const [relationshipType, setRelationshipType] = useState<string>('')
+  const [email, setEmail] = useState<string>('')
+  const [phoneNumber, setPhoneNumber] = useState<string>('')
+  const [notes, setNotes] = useState<string>('')
+  const [errorMessage, setErrorMessage] = useState<string>('')
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
same here - should we be commenting out this line if we're declaring variables that aren't being used?

Reply:
I am using the setErrorMessage so I put the line below in my return statement. So hopefully that's valid.

{<p className="show_error">{showError}</p>}